app.js

```diff
@@ -87,14 +95,13 @@ document.querySelector('#book-form').addEventListener('submit', (e) => {
 
   // Validate
   if (titleI === '' || authorI === '') {
-    showAlert('title and author must not be empty', 'success');
+    UI.showAlert('title and author must not be empty', 'success');
   } else {
-    const books = getBooks();
-    books.push({
-      title: titleI,
-      author: authorI,
-    });
-    setBooks(books);
+    const books = Store.getBooks(); // get books from local storage
+    const book = new Book(titleI, authorI); // new instance of Book
+    books.push(book); // push new book into books array
```

Comment:
- [ ] [Optional] I would advise you to use concat instead of push, this is because slice mutates the array whereas the splice method doesn't, changing array itself is a best practice instead of mutating it, because it will help you to not waste your resources